scripts/package-builder.js

```diff
@@ -112,10 +112,10 @@ npm install @leafygreen-ui/${PACKAGE_LC}
 `;
 
 const rootFile = `
-import React from 'react'
+import React from 'react';
 
 export default function ${PACKAGE_UC}({}) {
-  return(<div>your content here</div>)
+  return(<div>your content here</div>);
```

Comment:
while you're at it could you add a space after `return` and remove the parens lol

Reply:
Hah sure